src/widgets/modals/ByProgramModal.js

```diff
@@ -48,13 +49,19 @@ const modalProps = ({ dispatch, program, orderType }) => ({
         isEmergency,
       } = item;
 
+      const thisStoresTags = UIDatabase.getSetting(SETTINGS_KEYS.THIS_STORE_TAGS);
+      const maxLinesForOrder = program.getMaxLines?.(item.name, thisStoresTags);
+
       const mosText = `${maxMOS}: ${itemMOS}`;
       const thresholdText = `${threshMOS}: ${itemThreshMOS}`;
-      const maxOrdersText = isEmergency
-        ? programStrings.emergency_orders
-        : `${maxOPP}: ${maxOrders}`;
+      const maxItemsText =
+        maxLinesForOrder && maxLinesForOrder !== Infinity
+          ? `${programStrings.max_items}: ${maxLinesForOrder}`
+          : '';
+      const emergencyText = `[${programStrings.emergency_order}]  ${maxItemsText}`;
+      const maxOrdersText = isEmergency ? emergencyText : `${maxOPP}: ${maxOrders}`;
 
-      return `${mosText} - ${thresholdText} - ${maxOrdersText}`;
+      return `${maxOrdersText} - ${mosText} - ${thresholdText}`;
     },
   },
   period: {
```

Comment:
Line 70/77 below has `emergency_orders`, but you changed the translation key to `emergency_order`

Reply:
yup 99e5ad cheers